core/src/main/java/org/mule/expression/transformers/BeanBuilderTransformer.java

```diff
@@ -114,7 +114,7 @@ public Object transformMessage(MuleMessage message, String outputEncoding) throw
             if (!argument.isOptional() && value == null)
             {
                 throw new TransformerException(CoreMessages.expressionEvaluatorReturnedNull(
```

Comment:
if expression evaluators are gone, why do we still need a expression evaluator related message?